manifests/generated/kubevirt-ssp-operator.vVERSION.clusterserviceversion.yaml

```diff
@@ -78,6 +78,19 @@ spec:
           - update
```

Comment:
Please ignore all files under the `cluster/` and `manifests/` directories EXCEPT for this one